src/main.rs

```diff
@@ -411,6 +408,10 @@ enum Commands {
         #[clap(short, long)]
         compression_type: Option<String>,
         #[clap(short, long)]
+        min_record_size: Option<usize>,
+        #[clap(short, long, default_value_t = 16384)]
+        max_record_size: usize,
```

Comment:
should probably verify that `max_record_size >= min_record_size`

Reply:
Fair point -- in general this tool is not super friendly when it comes to validation etc, but not reason not to improve it.  Added a check.